.github/workflows/ci.yml

```diff
@@ -60,6 +60,7 @@ jobs:
         username: ${{ secrets.DOCKERHUB_USERNAME }}
         password: ${{ secrets.DOCKERHUB_TOKEN }}
     - name: Login to GitHub Container Registry
+      if: github.repository == 'bufbuild/plugins'
```

Comment:
Made steps of existing workflows that can't be run on forks behind a conditional.